### PR TITLE
Contribution design tweaks

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?

This is the first version of a much simpler a clear design of the checkout page for monthly contributions.

## Changes
* Created a new skin for the monthly contribution form.
* Added the specific files to support the new design.

### Before

monthlybefore

### After

monthlyafter

## Tested  in

Tested the Stripe and Paypal supporter flow in IE10 in Dev and is working ✅

### CODE REVIEW

**Comment by rupertbates** [comment]
Your screenshots seem to have got lost

File: frontend/app/views/joiner/form/monthlyContribution.scala.html [hunk 1]
```diff
@@ -58,11 +58,11 @@
         }());
     </script>
 
-    <main class="page-content skin-checkout-b skin-checkout-paypal">
+    <main class="page-content skin-checkout-b skin-checkout-paypal skin-monthly-contribution">
```
Comment (rupertbates) [inline frontend/app/views/joiner/form/monthlyContribution.scala.html]: Do we still need the other 2 skins here? skin-checkout-b & skin-checkout-paypal

File: frontend/app/views/joiner/form/monthlyContribution.scala.html [hunk 2]
```diff
@@ -99,54 +97,51 @@ <h2 class="form-panel__headline">Name</h2>
...
-                        <fieldset class="fieldset js-payment-methods">
-                            <div class="fieldset__heading">
-                                <h2 class="fieldset__headline fieldset__headline--payment">Payment methods</h2>
-                                <p class="security-note">@fragments.inlineIcon("lock", List("icon-inline--small", "icon-inline--top")) Secure</p>
+                    <div class="form-panel__content">
+                        <div class="fieldset__fields  js-form-panel-payment form-panel--no-margin">
+
+                            <div class="js-monthly-contribution">
+                                <div class="form-field js-monthly-contribution-section">
+                                    <label class="label" for="monthly-contribution">Your monthly contribution <span class="text-note text-note--monthly-contribution">(includes VAT)</span></label>
```
Comment (rupertbates) [inline frontend/app/views/joiner/form/monthlyContribution.scala.html]: As discussed we are not charging VAT

File: frontend/app/views/joiner/form/monthlyContribution.scala.html (hunk 1 above)
Comment (svillafe) [inline frontend/app/views/joiner/form/monthlyContribution.scala.html]: Yes, the `skin-monthly-contribution` loads the exception assuming that there are the other two skins applied. A new design will be introduced soon and probably we can drop these skins from this form.

**Comment by svillafe** [comment]
## Tested in PROD
| Browser | Supporter Stripe checkout | Supporter Paypal checkout | Stripe Monthly Contribution | Paypal Monthly Contribution |
|---------|-----------------|-----------------|----------------------|----------------------|
| IE      |                    |                      |                       |                         |
| Chrome  |   ✅                   |                           |                         |                       |

**Pull request merged**